src/404.html

```diff
@@ -88,15 +88,15 @@
 		<main id="main" class="l-interiorPage">
 			<header class="u-bg-gray-500 u-col-span-full">
 				<cap-page-header heading="Not Found.">
-					<p class="u-text-white u-text-serif">
+					<p class="u-text-white u-text-serif" style="visibility: hidden">
```

Comment:
Did you explore adding an additional utility class here, (for example, `class=u-text-white-u-text-serif u-visibility-hidden`) rather than inline style? That would prevent you from needing to resort to using `!important` in the component itself.  

While testing this in an incognito window, I did not visually perceive a difference between a slotted `p` element with this utility class and an inline style declaration.

Reply:
Hmm. I wasn't having any luck with a utility class getting picked up in time. Let me try again, maybe I had a typo.

Reply:
If for whatever reason I was not perceiving the difference between adding a utility class to one paragraph, and leaving in the inline style for other paragraphs (and watching for a change to affect the one paragraph and not others), that's okay — it's super small!

Reply:
Yeah, I don't think that works:
- added `u-visibility-hidden` to the `<p>`s
- added to utilities.css:
```
.u-visibility-hidden {
	visibility: hidden;
}
```
- switched cap-page-header.js to:
```
::slotted(p) {
    visibility: visible;
```

And, the pargraphs are visible on load (presumably while css is loading), and are hidden after load (presumably because the slotted p selector is less specific than the utility class).

Do you have a different strategy in mind than the above? Did I do that wrong?

Reply:
I don't think you did this wrong, at all! 

It's a strange situation, and yes — you are absolutely right that class selectors have a slightly greater specificity than element selectors. I'm sure there may be an approach here that would allow us to avoid the !important but I think using it once for an odd edge case like this is absolutely fine.